### DISCUSSION TITLE
Blazorise.Charts - the type or namespace name 'Blazorise' could not be found

### DISCUSSION THREAD

**Discussion by mindaugas-epoint** [Megabit/Blazorise#2170] [Q&A] [answered]
Hello,
I want to test Blazorise.Charts in my app, but getting missing reference error:<...>_Imports.razor(7,7): Error CS0246: The type or namespace name 'Blazorise' could not be found (are you missing a using directive or an assembly reference?) (CS0246)

I've completed all the installation steps as described here - https://blazorise.com/docs/extensions/chart/
![Screen Shot 2021-04-07 at 12 59 26](https://user-images.githubusercontent.com/50573568/113849228-f15f1000-97a1-11eb-81c7-08c9e46d8574.png)

Have I missed something?

Visual Studio 2019 for Mac

**Comment by stsrki** [comment, marked as answer]
Do you use the same version of .Net Core?

**Reply by mindaugas-epoint** [reply]
I am on .Net.Core 3.1. So I have to use .Net 5.0 to use Blazorise.Charts latest version, right?

**Reply by stsrki** [reply]
Yes, starting from 0.9.3 only .Net 5 is supported. 

https://blazorise.com/news/release-notes/093/#migration-

**Reply by mindaugas-epoint** [reply]
Thank you!
